plugins/module_utils/ilo_redfish_utils.py

```diff
@@ -231,3 +608,1407 @@ def set_wins_registration(self, mgrattr):
         if not response['ret']:
             return response
         return {'ret': True, 'changed': True, 'msg': "Modified %s" % mgrattr['mgr_attr_name']}
+
+    def verify_drive_count(self, raid_details, logical_drives_count):
```

Comment:
Is this going to remain specific to Smart Storage?

Reply:
This function is used internally by the verify_logical_drives functionality. So, it is SmartStorage specific.